src/compile_config.h

```diff
@@ -21,19 +21,26 @@
 */
 
 #ifdef OTHER_BOARD_WITH_CC1101
-#define CMP_CC1101
+	#define CMP_CC1101
 #endif
 #ifdef ARDUINO_ATMEGA328P_MINICUL
-#define CMP_CC1101
+	#define CMP_CC1101
 #endif
 
 // Get compatibility with arduino ide and visualmicro
 #ifdef ARDUINO_AVR_ICT_BOARDS_ICT_BOARDS_AVR_RADINOCC1101
-#define ARDUINO_RADINOCC1101
+	#define ARDUINO_RADINOCC1101
 #endif
 
 #ifdef ARDUINO_RADINOCC1101
-#define CMP_CC1101     
+	#define CMP_CC1101
+#endif
+
+#if defined(ARDUINO_MAPLEMINI_F103CB) || defined(STM32F103xB)          // STM32 F103CBT6

```

Comment:
Ich habe nirgends gefunden, wo STM32F103xB definiert wird.


Die Definition von  ARDUINO_MAPLEMINI_F103CB habe ich gefunden:
https://danieleff.github.io/STM32GENERIC/build_macros/#macros-specific-to-board


Für mich stellt sich aber schon die Frage, ob man hier wirklich Maple_MINI auf Basis spezifischer Boards definiert oder nicht bezüglich der STM32 Anteile mit "ARDUINO_ARCH_STM32" arbeiten sollte.

MAPLE_Mini scheint mir eine Implementierung des Boards ARDUINO_MAPLEMINI_F103CB zu sein und das Board gehört zur Archiektur ARDUINO_ARCH_STM32

Reply:
Ich glaube boards.txt wird nur von der Arduino IDE verwendet aber nicht von Plattform IO

https://docs.platformio.org/en/latest/platforms/creating_board.html

Seltsam ist aber, dass die Macros aus der STM Bibliothek nicht funktionieren.
Das würde ja bedeuten, dass die nicht eingebunden sind, wenn compile_config.h compiliert wird.

Das klappt beim ESP und bei AVR ja auch, daher würde ich es gerne beim STM32 sehr ähnlich macheb

Reply:
Ich habe nun noch einmal getestet und habe es nur mit
`#if defined(ARDUINO_MAPLEMINI_F103CB)  // STM32 F103CBT6`
versucht, es klappt doch. Keine ahnung wie ich darauf kam. Dann wäre doch die Frage hier geklärt oder?